pkg/statistics/handle/storage/read.go

```diff
@@ -625,6 +625,10 @@ func loadNeededColumnHistograms(sctx sessionctx.Context, statsHandle statstypes.
 			return nil
 		}
 		colInfo = tblInfo.Meta().GetColumnByID(col.ID)
+		if colInfo == nil {
```

Comment:
can you change the PR title, seems we are fixing some corner case of stats gathering

Reply:
OK，Updated.